source/ShipJumpNavigation.cpp

```diff
@@ -42,8 +42,6 @@ ShipJumpNavigation::ShipJumpNavigation(const Ship &ship)
 // Calibrate this ship's jump navigation information, caching its jump costs, range, and capabilities.
 void ShipJumpNavigation::Calibrate()
 {
-	if(!ship)
```

Comment:
Why remove this?

Reply:
Because it isnt required anymore but feel free to readd it if you want